components/ocf-host-application/ApplicationForm.js

```diff
@@ -0,0 +1,705 @@
+import StyledInputGroup from '../StyledInputGroup';
+import StyledLink from '../StyledLink';
+import StyledTextarea from '../StyledTextarea';
+import { H1, H4, P } from '../Text';
+
+import OCFPrimaryButton from './OCFPrimaryButton';
+
+const createCollectiveMutation = gqlV2/* GraphQL */ `
+  mutation CreateCollective(
+    $collective: CollectiveCreateInput!
+    $host: AccountReferenceInput
+    $user: UserCreateInput
+    $message: String
+  ) {
+    createCollective(collective: $collective, host: $host, user: $user, message: $message) {
+      id
+      slug
+      host {
+        id
+        slug
+      }
+    }
+  }
+`;
+
+const messages = defineMessages({
+  locationLabel: {
+    id: 'OCFHostApplication.location.label',
+    defaultMessage: 'Your Location',
+  },
+  nameLabel: {
+    id: 'OCFHostApplication.name.label',
+    defaultMessage: 'Your Name',
+  },
+  emailLabel: {
+    id: 'OCFHostApplication.email.label',
+    defaultMessage: 'Your email address',
+  },
+  initiativeNameLabel: {
+    id: 'OCFHostApplication.initiativeName.label',
+    defaultMessage: 'What is the name of your initiative?',
+  },
+  slugLabel: {
+    id: 'OCFHostApplication.slug.label',
+    defaultMessage: 'What URL would you like?',
+  },
+  initiativeDurationLabel: {
+    id: 'OCFHostApplication.initiativeDuration.label',
+    defaultMessage: 'How long has your initiative been running?',
+  },
+  totalAmountRaisedLabel: {
+    id: 'OCFHostApplication.totalAmountRaised.label',
+    defaultMessage: 'If you have begun fundraising, how much money have you raised so far?',
+  },
+  totalAmountToBeRaisedLabel: {
+    id: 'OCFHostApplication.totalAmountToBeRaised.label',
+    defaultMessage: 'How much money do you want to fundraise?',
+  },
+  expectedFundingPartnerLabel: {
+    id: 'OCFHostApplication.expectedFundingPartner.label',
+    defaultMessage: 'Who do you expect to fund you?',
+  },
+  initiativeDescriptionLabel: {
+    id: 'OCFHostApplication.initiativeDescription.label',
+    defaultMessage: 'What does your initiative do?',
+  },
+  missionImpactExplanationLabel: {
+    id: 'OCFHostApplication.missionImpactExplanation.label',
+    defaultMessage: 'Please explain how your initiative furthers one or more of our mission impact areas:',
+  },
+  websiteAndSocialLinksLabel: {
+    id: 'OCFHostApplication.websiteAndSocialLinks.label',
+    defaultMessage: 'Website and / or social media links:',
+  },
+  additionalInfoLabel: {
+    id: 'OCFHostApplication.additionalInfoLabel.label',
+    defaultMessage: 'Anything you would like to add?',
+  },
+});
+
+const initialValues = {
+  location: '',
+  name: '',
+  email: '',
+  initiativeName: '',
+  slug: '',
+  initiativeDuration: '',
+  totalAmountRaised: '',
+  totalAmountToBeRaised: '',
+  expectedFundingPartner: '',
+  initiativeDescription: '',
+  missionImpactExplanation: '',
+  websiteAndSocialLinks: '',
+  additionalInfo: '',
+  termsOfServiceOC: false,
+  termsOfServiceOCF: false,
+};
```

Comment:
Formik supports nested values. A cleaner way to write that wouldn't require formatting the payload later would be something like:
```es6
const initialValues = {
  collective: {
    name: '',
    slug: '',
  },
  user: {
    name: '',
    email: '',
  }, ...
};
```

To use them, you then pass something like `collective.slug` as a name for the field.